invesalius/data/elfin.py

```diff
@@ -24,17 +24,19 @@ def Initialize(self):
     def Run(self):
         return self.cobot.ReadPcsActualPos()
 
-    def SendCoordinates(self, target, type=const.ROBOT_MOTIONS["normal"]):
+    def SendCoordinates(self, target, motion_type=const.ROBOT_MOTIONS["normal"]):
```

Comment:
Thanks for changing this variable name, `motion_type` seems to me more precise than just `type`.